### PR TITLE
111 tests for calculating and inserting total values into csv

### PR DESCRIPTION
**Pull request by ChaseStauts**
references #111 The current iteration of the test suite for given calculating totals, there might be better ways to implement but this is how I got it to work. Let me know if there needs to be any changes.

### CODE REVIEW

**Review by PaulEllis2001 — APPROVED** [review]

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ExternalStorageConfigurationManager" enabled="true" />
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" project-jdk-name="17" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="17" project-jdk-type="JavaSDK">
```
Comment (BreeLatimer) [inline .idea/misc.xml:4]: Is this supposed to be here?

Comment (ChaseStauts) [reply .idea/misc.xml:4]: I don't remember adding it, let me see if I can revert that

File: src/jvmTest/kotlin/TotalDataCalcTests.kt [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        servCellTotal = 0.0
+        travelCellTotal = 0.0
+        completeTotal = 0.0
+        expectedToString = " "
+        actualToString = ""
+        // return tempFile
+    }
+
+    @Test
+    fun totalData_NormalDataFormat() {
```
Comment (BreeLatimer) [inline src/jvmTest/kotlin/TotalDataCalcTests.kt:57]: For this test and for other tests can we change the name of the tests to be the `given_when_then` naming format? :)

Comment (ChaseStauts) [reply src/jvmTest/kotlin/TotalDataCalcTests.kt:57]: Gotcha

Comment (ChaseStauts) [reply src/jvmTest/kotlin/TotalDataCalcTests.kt]: Pushed changes @BreeLatimer

**Review by BreeLatimer — APPROVED** [review]
Looks great! :)

**Pull request merged**